projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+{
+  "advancement.ratlantis.defeat_rat_baron.desc": "击败老鼠男爵",
+  "advancement.ratlantis.defeat_rat_baron.title": "死亡或荣耀",
+  "advancement.ratlantis.dutchrat_wheel.desc": "击败飞翔的荷兰鼠",
+  "advancement.ratlantis.dutchrat_wheel.title": "LEDLE LEDLE LEDLE LEEEE",
+  "advancement.ratlantis.feral_ratlantean.desc": "遇到鼠西洲过去的高贵居民如今的残态",
+  "advancement.ratlantis.feral_ratlantean.title": "追本溯源",
+  "advancement.ratlantis.gem_of_ratlantis.desc": "用鼠套花瓣和绿宝石来制作鼠西洲宝石",
+  "advancement.ratlantis.gem_of_ratlantis.title": "近乎圣洁",
+  "advancement.ratlantis.ghost_pirat.desc": "遭遇幽灵鼠盗，久逝水手的被诅咒的灵魂",
+  "advancement.ratlantis.ghost_pirat.title": "吓我一跳",
+  "advancement.ratlantis.marbled_cheese_golem.desc": "击败鼠西族战机",
+  "advancement.ratlantis.marbled_cheese_golem.title": "人机对决（或鼠机对决）",
+  "advancement.ratlantis.marbled_cheese_golem_core.desc": "合成鼠西族战机核心！现在，将鼠西族战机头颅放在顶部，并用大理石纹奶酪包裹核心！（任意大理石纹奶酪，做成铁傀儡的形状）",
+  "advancement.ratlantis.marbled_cheese_golem_core.title": "我们可以重构……",
+  "advancement.ratlantis.neoratlantean.desc": "击败新·鼠西族",
+  "advancement.ratlantis.neoratlantean.title": "但伙计，我数学不好吗",
```

```suggestion
  "advancement.ratlantis.neoratlantean.title": "但天哪，我的数学可真差",
```
The risks I took were calculated, but man, am I bad at math.
与`advancement.ratlantis.vial_of_sentience.title`连起来翻译。